src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/IcoNumberTests.cs

```diff
@@ -73,7 +73,7 @@ public void check_orchestrator_builds_with_ico_number()
 
             var viewModel = response.Result;
 
-            Assert.AreEqual(UKRLPLegalName, viewModel.UkrlpLegalName);
+            Assert.AreEqual(UKRLPLegalName, viewModel.ApplyLegalName);
```

Comment:
Not sure this is correct and may need to revert this change. You're now checking the legal name within Apply is equal to that from UKRLP? Is it not possible that it could have changed within UKRLP over time?

Reply:
We need to use only ApplyLegalName as common property in RoatpGatewayPageViewModel. And it is the only one populated by PopulatePageCommonDetails.
If someone needs UkrlpLegalName in their particular ViewModel, they can deal with it in their implementation.